enterprise/backend/src/metabase_enterprise/gsheets/api.clj

```diff
@@ -73,23 +73,27 @@
 
 (set! *warn-on-reflection* true)
 
-(defn error-response-in-body
+(defn- loggable-response [hm-response]
```

Comment:
maybe more idiomatic/readable would be

```
(as-> hm-response r
 (if (sequential? r) (last r) r)
 (or (:ex-data r) r))
 (select-keys r [:status :body]))
```

Reply:
I see now you didn't write this in the first place though so definitely non-blocking.